docs/install/knative-with-any-k8s.md

```diff
@@ -398,6 +398,18 @@ To learn more about the Apache Kafka channel, try [our sample](../eventing/sampl
 
 {{< /tab >}}
 
+{{% tab name="Google Cloud Pub/Sub Channel" %}}
+
+1. Install the Google Cloud Pub/Sub Channel:
+
+   ```bash
+   kubectl apply --filename {{< artifact repo="knative-gcp" file="cloud-run-events.yaml" >}}
```

Comment:
```suggestion
   kubectl apply --filename {{< artifact org="google" repo="knative-gcp" file="cloud-run-events.yaml" >}}
```

Reply:
Do we need this for the source too?

Reply:
Done!
There are a couple of in-flight PRs to update the sources examples.  https://github.com/knative/docs/pull/2125/ and https://github.com/knative/docs/pull/2118. 
Hopefully that should go in soon

Reply:
BTW, it seems that we are telling folks to install the nightly instead of the most recent release? Was that the intention @mattmoor?